stats/stats.go

```diff
@@ -98,7 +101,10 @@ func (s *InPayload) IsClient() bool { return s.Client }
 
 func (s *InPayload) isRPCStats() {}
 
-// InHeader contains stats when a header is received.
+// InHeader contain stats when the header is received.
+// 
+// First event in the server side event sequence.
+// Follows last OutPayload for server side events.
```

Comment:
Same thing. Can start in same line. `It is the first event at the server after receiving the RPC. It is followed by `OutPayload` server event.`